package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "bootstrap": "3.3.7",
     "jquery": "3.2.1",
+    "jquery.hotkeys": "^0.1.0",
     "moment": "^2.18.1",
```

Comment:
please use exact versions (i forgot to do so in moment too), and bump patch

Reply:
Done 👍